prompts/natbot/prompt.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
i would not add this one - its a bit out of date and im not sure it works well. we chould check that it works well, but thats a longer discussion so for now i would just leave out